### PR TITLE
Parameter: Handle empty vector with class such as as.character(c()) correctly

### PR DESCRIPTION
**Pull request by hideaki**
# Description
Handle empty vector with class such as as.character(c()) correctly as a parameter value in generating SQL/MongoDB query.

# Checklist
Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/system.R [hunk 1]
```diff
@@ -339,7 +339,7 @@ js_glue_transformer <- function(expr, envir) {
     }
   }
 
-  if (is.null(val)) { # NULL in R is same as empty vector. Print empty string.
+  if (length(val) == 0) { # Empty vector case. NULL in R is same as empty vector, but since is.null(as.character(c(0))) returns FALSE, this has to be done my length().
```
Comment (hidekoji) [inline R/system.R:342]: Do you mean "by length()" by "my length()"?

File: R/system.R [hunk 2]
```diff
@@ -469,7 +469,8 @@ sql_glue_transformer_internal <- function(expr, envir, bigquery=FALSE) {
...
```
Comment (hidekoji) [inline R/system.R:472]: Do you mean "by length()" by "my length()"?

File: R/system.R (hunk 1 above)
Comment (hideaki) [reply R/system.R]: That's right. Fixed it. Thanks!

File: R/system.R (hunk 2 above)
Comment (hideaki) [reply R/system.R]: Fixed it. Thanks!

**Comment by hideaki** [comment]
The jenkins test errors are the expected ones until we update test docker image.

**Pull request merged**